source/containers/Form/hooks/formActions.ts

```diff
@@ -4,6 +4,23 @@ import { FormReducerState } from "./useForm";
 import { validateInput } from "../../../helpers/ValidationHelper";
 import { evaluateConditionalExpression } from "../../../helpers/conditionParser";
 import { deepCopy } from "../../../helpers/Objects";
+import { PartnerInfo, User } from "app/types/UserTypes";
+import { Person } from "app/types/Case";
+
+function computePartnerPersonInfo(person: Person): PartnerInfo {
+  return {
+    partnerLastname: person.lastName,
+    partnerName: person.firstName,
+    partnerPersonalid: person.personalNumber,
+  };
+}
+
+function computePersonUserInfo(person: Person): User {
+  return {
+    firstName: person?.firstName,
+    lastName: person?.lastName,
+  };
+}
```

Comment:
This seems to only return properties with the same name, could we just `return person`? Or even remove this function altogether? Unless there's a reason to _only_ include these properties (in which case we should change the return type to `Partial<User>`).

Reply:
Removed the function.